lib/commons/color/get-foreground-color.js

```diff
@@ -63,6 +64,11 @@ function getForegroundColor(node, _, bgColor) {
     return null;
   }
 
+  if (fgColor.alpha === 0) {
+    const textShadowColors = getTextShadowColors(node, { minRatio: 0 });
```

Comment:
This might hurt performance a bit. Can you work with @straker to figure out if this needs any performance optimisations?

Reply:
He didn't think it would, since it's only happening when fgColor has an alpha of < 1, which is more rare.